frontend/index.html

```diff
@@ -1,8 +1,8 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" />
```

Comment:
Shouldn't we use `@fortawesome/react-fontawesome` instead?

Reply:
Yes, we can use react-fontawesome and i will replace it.